tests/functional/test_integrity.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import random
+import datetime
+import time
+
+import pytest
+
+from core.test_run import TestRun
+from iotrace import IotracePlugin
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine, VerifyMethod
+from test_utils.size import Unit, Size
+
+
+runtime = datetime.timedelta(seconds=20)
+
+
+def test_data_integrity_20s():
+    TestRun.LOGGER.info("Testing data integrity")
+    iotrace: IotracePlugin = TestRun.plugins['iotrace']
+    for disk in TestRun.dut.disks:
+        with TestRun.step("Start tracing"):
+            iotrace.start_tracing([disk.system_path])
+        with TestRun.step("Run test workloads with verification"):
+            setup_workload(disk.system_path)
```

Comment:
I'd also pass here the runtime parameter.

Generally I think the setup_workload() function should be more generic  (although I don't know what other fio parameters would make sense to paramterize in python function here). Then it maybe would  sense to make the function correspond to some sort of workload type? So maybe it could be named "run_basic_rw_workload_verify". Then later these functions could be reused in other tests - e.g. performance.